src/modules/fw_pos_control_l1/FixedwingPositionControl.hpp

```diff
@@ -435,10 +463,19 @@ class FixedwingPositionControl final : public ModuleBase<FixedwingPositionContro
 
 		(ParamFloat<px4::params::NAV_LOITER_RAD>) _param_nav_loiter_rad,
 
-		(ParamFloat<px4::params::FW_TKO_PITCH_MIN>) _takeoff_pitch_min
+		(ParamFloat<px4::params::FW_TKO_PITCH_MIN>) _takeoff_pitch_min,
 
-	)
+		//Speed mode params
+		(ParamFloat<px4::params::FW_WIND_ARSP_SC>) _param_fw_wind_arsp_sc,
 
+		(ParamInt<px4::params::FW_SPD_MODE_SET>) _fw_spd_mode_set,
+		(ParamFloat<px4::params::FW_ALT_MIN>) fw_alt_min,
+		(ParamFloat<px4::params::FW_ALT_ERR_U>) fw_alt_err_u,
+		(ParamFloat<px4::params::FW_ALT_ERR_O>) fw_alt_err_o,
```

Comment:
Nitpicking here, but the leading underscore seems to be inconsistent with global variables

Reply:
yes ofc :+1: , I've changed it.